R/get_habitat.R

```diff
@@ -1,14 +1,16 @@
 #' @title Get habitat Values
 #' @source IP-117068
-#' @param year TODO
-#' @param month TODO
-#' @param inchannel_habitat_fry TODO
-#' @param inchannel_habitat_juvenile TODO
-#' @param floodplain_habitat TODO
-#' @param sutter_habitat TODO
-#' @param yolo_habitat TODO
-#' @param north_delta_habitat TODO
-#' @param south_delta_habitat TODO
+#' @description Returns the appropriate habitat for spawning and rearing based on habitat inputs and a given simulation year and month
+#' @details See \code{\link{params}} for details on parameter sources
+#' @param year The current simulation year, 1-20
+#' @param month The current simulation month, 1-8
+#' @param inchannel_habitat_fry Variable describing fry inchannel rearing habitat
```

Comment:
describing is a bit weird word choice. Maybe something like "3 dimensional array [watersheds, months, years] representing fry inchannel rearing habitat in square meters"

please modify the others as well

Reply:
Fixed this one. Should I modify all others to match this general structure?